data/en/systemcacheclear.json

```diff
@@ -6,11 +6,41 @@
 	"related": [],
 	"description": "Clears all caches or the specified cache",
 	"params": [
-		{"name": "cacheName", "description": "", "required": false, "default": "", "type": "string", "values": []}
+		{
+			"name": "cacheName", 
+			"description": "Name of the cache you want to clear. If not specified it will clear all caches.", 
+			"required": false, 
+			"default": "", 
+			"type": "string", 
+			"values": [
+				"all (clear all caches)",
```

Comment:
`values` should only contain the values themselves, not descriptions.

Reply:
Okay, but what if a value can contains aliases and means the same?  
E.g.  template or page  means both cleaing the template cache.

Reply:
Then they both should be included, because IDE plugins can provide both values for autocomplete.

Reply:
So how you recommend to write this? As seperate values without description? But than the information is lost that page/template means the same.

Reply:
You can add a note to the param's description.